docs/source/development/debugging.md

```diff
@@ -1,10 +1,8 @@
 # Debugging
 
-:::note
-
+``` {note}
 Our debugging documentation has moved. Please see our existing guides:
-
-:::
+```
 
 * [Debugging a Kedro project within a notebook](../notebooks_and_ipython/kedro_and_notebooks.md#debugging-a-kedro-project-within-a-notebook) for information on how to launch an interactive debugger in your notebook.
```

Comment:
Can we split this into a few section? I would love to highlight the `%load-node` debug magic

Reply:
What split would you propose? The line magic itself has it's own section above, and the debug workflow with and without it are almost identical

Reply:
Maybe `### `%load_node` line magic`? I'd also re-order it, I think debugger & %load_node should go up and the hooks or pdb/%debug should be lower.